src/ensembl/src/content/app/browser/browserHelper.test.ts

```diff
@@ -62,18 +61,18 @@ describe('browserHelper', () => {
         value: faker.random.number()
       }
     };
-    const mockValidationResult = createRegionValidationResult(); // all error messages are undefined by default
+    const mockValidationMessages = createRegionValidationMessages(); // all error messages are undefined by default
 
     test('returns all error messages as undefined', () => {
```

Comment:
`undefined` no longer describes the code correctly?

(here and elsewhere in the file)

Reply:
Fixed it